.azure-pipelines/gpu-tests.yml

```diff
@@ -51,7 +51,7 @@ jobs:
     - bash: |
         python -c "fname = 'requirements/extra.txt' ; lines = [line for line in open(fname).readlines() if 'horovod' not in line] ; open(fname, 'w').writelines(lines)"
         pip install fairscale>=0.3.4
-        pip install "deepspeed>=0.4.0, !=0.4.4"  # FIXME: bug with 0.4.4
+        pip install "deepspeed>=0.4.3, !=0.4.4"  # FIXME: bug with 0.4.4
```

Comment:
@SeanNaren Thomas did a similar fix in #8674?

Reply:
Yeah I think we'll overwrite this PR's with Thomas's as we try to fix the issue! Whichever gets merged first :)

Reply:
It's merged. I think the rebase would work?

Reply:
oh i thought you were talking about https://github.com/PyTorchLightning/pytorch-lightning/pull/8655 haha